File: Source/Fantasy.Jobs/CallTemplateParameter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.XSerialization;

namespace Fantasy.Jobs
{
    [XSerializable("parameters", NamespaceUri = Consts.XNamespaceURI)]
    internal class CallTemplateParameter : IConditionalObject
    {
        [XAttribute("itemCategory")]
        public string ItemCategory = null;

        [XAttribute("include")]
        public string Include = null;

        [XAttribute("condition")]
        private string _condition = null;
        string IConditionalObject.Condition
        {
            get
            {
                return this._condition;
            }
            set
            {
                this._condition = value;
            }
        }

        [XAttribute("propertyName")]
        public string PropertyName = null;
        [XAttribute("value")]
        public string Value = null;
    }
}

File: Fantasy.Jobs/src/fantasy/jobs/CallTemplateParameter.java
package fantasy.jobs;

import fantasy.xserialization.*;

@XSerializable(name = "parameters", namespaceUri = Consts.XNamespaceURI)
public class CallTemplateParameter implements IConditionalObject
{
	@XAttribute(name = "itemCategory")
	public String ItemCategory = null;

	@XAttribute(name = "include")
	public String Include = null;

    @XAttribute(name = "condition")
	private String _condition = null;
	public String getCondition()
	{
		return this._condition;
	}
	public void setCondition(String value)
	{
		this._condition = value;
	}

	@XAttribute(name = "propertyName")
	public String PropertyName = null;
	
	@XAttribute(name = "value")
	public String Value = null;
}